### ISSUE TITLE
Feature Request: Allow limiting arm options

### ISSUE THREAD

**Issue by brendon** [EHylands/homebridge-boschcontrolpanel_bgseries#12]
Hi @EHylands, I thought I'd start a new thread for this.

The konnected.io homebridge plugin supports disabling certain away states as an option. In my case I don't use 'Home' or 'Night' modes (as presented by your plugin) and would prefer to just have an on/off switch in Home. Solution 3000 just has Away, and two forms of Stay (1 & 2) that can be configured separately, then just 'Off'.

Being able to disable certain states at the plugin level would be quite useful. All good if it's too complicated though :D

**Comment by EHylands** [comment]
It is a easy option. Recently wrote a plugin for qolsys iq panels with disabled night mode.

Only problem I foresee is that any changes to the security panel accessory options will require user to remove and add accessory again. All linked automations would have to be linked again in the process

**Comment by brendon** [comment]
That's true. I'd imagine you'd make upgrading easy by leaving everything on by default, so only those that venture to either configure this explicitly when they first install the plugin, or who twiddle the settings later on purpose would encounter this issue?

**Comment by brendon** [comment]
Hi @EHylands, just wondering if you're still interested in looking at this again? I was investigating seeing if there's a way to create a virtual toggle within homebridge to do the same thing.
